force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -163,9 +186,10 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer generateRandomNumber() {
         // Use Math class to generate number between 50 and 100
-
+        Double random = math.random();
```

Comment:
Consider using the following formula instead to generate a random number (leaving the `Math.random()` method inside the formula): 
`Integer randomNumber = (Integer) (Math.random() * (max - min + 1)) + min;`